orchestrator/nodes/dockerComposeRemoteNode/index.js

```diff
@@ -48,13 +48,20 @@ class DataHandler extends RemoteNode {
       }
 
       this.client.networks().list().then((results) => {
-        for (let result of results) {
-          let name = result.Name.match(/.+?_flowbroker/);
-          if (name) {
-            this.targetNetwork = name;
-            return resolve(name);
+        let network = process.env.FLOWBROKER_NETWORK;
```

Comment:
I think that should be interesting add this environment variable on 'config.js' file.
When env variables are just used in somewhere in the code become difficult to have an overview that which environment variables are requested.

Reply:
What do you think about it now?

Reply:
seems fine.